Automattic/wp-calypso
# Build: Move FilterWarningsPlugin to calypso-build's sass config

#### Changes proposed in this Pull Request

Move `FilterWarningsPlugin` to calypso-build's sass config

Since we're invoking `mini-css-extract-plugin-with-rtl` from `calypso-build`'s `sass.js` now, I think it makes sense to also suppress the order warnings there.

#### Testing instructions

The usual:

```
npm run distclean
npm ci
npm start
```

Verify that styling doesn't look odd, and that `mini-css-extract-plugin` doesn't throw any `Conflicting order` warnings on the node console.

Fixes #31367
